packages/flutter/test/widgets/binding_test.dart

```diff
@@ -205,7 +205,7 @@ void main() {
     FlutterError.onError = oldHandler;
     const String toMatch = '...     Normal element mounting (';
     expect(toMatch.allMatches(filteredStack)?.length, 1);
-  });
+  }, skip: kIsWeb); // The `flutter run` and `flutter test` commands produce different output.
```

Comment:
This looks like a regression worth fixing.

Reply:
This is code is testing the output of flutter test, which is different than the output of flutter run. I tried to update the stack trace parsing to work better for flutter run, but instead I could revert all of the changes and leave that as a follow up - might be easier